Content.Shared/Silicons/Borgs/Components/ItemBorgModuleComponent.cs

```diff
@@ -14,15 +15,27 @@ public sealed partial class ItemBorgModuleComponent : Component
     /// <summary>
     /// The items that are provided.
     /// </summary>
-    [DataField("items", customTypeSerializer: typeof(PrototypeIdListSerializer<EntityPrototype>), required: true)]
+    [DataField("items", customTypeSerializer: typeof(PrototypeIdListSerializer<EntityPrototype>))] // Frontier: removed
```

Comment:
this should probably be `removed "required: true"`?

Reply:
it exists on frontier, dont want a deltav comment for correcting a frontier comment :trollface: